app/views/users/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="well well-sm">
+  <div>
+  <h5><%= current_user.email %></h5>
+  </div>
+</div>
+
+<p>
+<div class="col-md-8">
+  <%= form_for [@user, @item] do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name, placeholder: "New List"  %>
+    <%= f.submit "Save" %>
+  <% end %>
+    <div class="form-group" >
+    </div>
+  </div>
+</p>
+
+<div class="container">
+  <div class="col-md-8">
+    <table>
+      <% @items.each do |i| %>
+        <tr>
+          <td><%= @item.name %></td>
+          <td><%= link_to @items_path %></td>
```

Comment:
This is giving me a title of: ```
[#<Item id: nil, name: nil, user_id: nil, created_at: nil, updated_at: nil>, nil]```

I get what you had me do.  I get that I was calling something that doesn't exist.  But now, I don't get why it's telling me that the item and the user_id are nil.  I tried pulling the information up in the console but the error message said ```VM2365:1 Uncaught ReferenceError: item is not defined``` and when I search for that it gives me javascript error... so now I'm really at a loss on how to approach this.